share/revng/test/tests/analysis/ImportFromCAnalysis/struct-with-conflicting-annotations/expected-error.txt

```diff
@@ -2,6 +2,5 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
-import-from-c: Multiple conflicting annotation values found: '8' and '16'.
-
+import-from-c:7:16: Multiple conflicting values ('8' and '16') were found for the 'field_start_offset:' annotation.
```

Comment:
what's `field_start_offset` annotation? that's not something the user ever sees. what they see is `_START_AT` and that's also what they should be seeing here.